api/src/models/ImageFile.ts

```diff
@@ -71,11 +72,9 @@ class ImageFile {
     }
 
     async ocr(): Promise<void> {
-        //TODO: update the following (derivativepath requires size now)
```

Comment:
Is this TODO really done? Just curious why the only changes in this file are the removal of comments and reorganization of imports... is this intentional or a merge artifact?